lib/applicationModel.js

```diff
@@ -186,15 +188,6 @@ class ApplicationModel {
     return schemaName;
   }
 
-  stripPackageName(schemaName) {
-    // If there is a dot in the schema name, it's probably an Avro schema with a fully qualified name (including the namespace.)
-    const indexOfDot = schemaName.lastIndexOf('.');
-    if (indexOfDot > 0) {
-      return { className: schemaName.substring(indexOfDot + 1), javaPackage: schemaName.substring(0, indexOfDot) };
-    }
-    return { className: schemaName, javaPackage: undefined };
-  }
-
```

Comment:
Moved to utility class since it's common for all.js and applicationModel.js. Could also be used in other places in the future.
Not sure whether or not it should live in scsLib.js since the comment in the file says, `// This contains functions that are common to both the all.js filter and the post-process.js hook`.

Reply:
My intention with scsLib.js was to have a place for code that could be shared between the filters (all.js) and any non-filters such as the pre and post hooks. So it would be fine to put that in scsLib.

Reply:
Okay, I'll move it there, thanks!